.env

```diff
@@ -42,6 +42,19 @@ TAG=local
 # The domain at which your production site is hosted.
 DOMAIN=islandora.dev
 
+# Set to "on" if your ISLE docker deployment is behind a reverse proxy
+REVERSE_PROXY=off
+
+# This list should be all the IPs in front of your Drupal docker container
+# this is used to pass the original client IP to the drupal container so
+# drupal/php is aware of who sent the original request
+# if you're not behind a reverse proxy, you probably do not need to edit these IPs
+# if you are behind a reverse proxy, most likely you can just replace FRONTEND_IP_1
+# with the IP address used on your front end // reverse proxy domain
+FRONTEND_IP_1=127.0.0.1/32
```

Comment:
You're welcome to open a new PR to simplify. FWIW I did it this way because the values are used in two places. One comma delimited used in docker compose(traefik), the other is multiple directives in buildkit // nginx directive.

Reply:
For the bits on nginx we can iterate on a list using confd, let's not worry about it for now though. If someone wants to use more than 3 ip's in the future we can sort it then.